media/videostitcher/create_cdn_key.go

```diff
@@ -14,74 +14,74 @@
 
 package videostitcher
```

Comment:
issue: remove all of the commented out code or un-comment it out.

Reply:
I needed to comment out some entire files because they now break due to the updated library version. I'll uncomment and update them in a separate PR.

Reply:
Commented-out files are still in PR.